app/views/posts/show.html.erb

```diff
@@ -12,7 +12,7 @@
       <h2><%= @post.title %></h2>
       <p><%= @post.content %></p>
     </div>
-     <%= cl_image_tag @post.user.profile.photo.key, width: 40, height: 40, gravity: :face, crop: :fill, class: "avatar dropdown-toggle", 'aria-haspopup': true, 'aria-expanded': false %>
+     <%= cl_image_tag @post.photos, width: 40, height: 40, gravity: :face, crop: :fill, class: "avatar dropdown-toggle", 'aria-haspopup': true, 'aria-expanded': false %>
```

Comment:
@awwebdev2020 , this photo should be avatar of the user. If they don't have avatar then we should do if else statement with
 if @post.user.profile.photo
else: give them a sample avatar